brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1045 Bug: Editor border: Z-index property turning blocks to white when bg color added to child of columns/container.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1045

### Screenshots

- Screencast of testing - https://share.getcloudapp.com/bLuBeXBR

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Container/Column.
- Use 3 column layout for both columns & container.
- Add bg color to parent column/container  i. e blue
- Add a bg color white to the child and hover on the child.
- On the hover block border should be visible, the block should be the same when clicking or hovering on the child.
- Check with box shadow, Border, Shape divider for container parent child.
- Check more test cases from [here](https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/f2ca0546-a16a-4a88-b6c9-403eeb96d480?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes)

### Checklist:
